frontend/src/static/js/components/webstatus-notfound-error-page.ts

```diff
@@ -50,75 +98,137 @@ export class WebstatusNotFoundErrorPage extends LitElement {
           line-height: 22.5px;
           word-wrap: break-word;
         }
+        #error-actions {
+          display: flex;
+          flex-wrap: wrap;
+          justify-content: center;
+        }
         #error-headline {
           color: #1d2430;
           font-size: 32px;
           font-weight: 700;
           word-wrap: break-word;
         }
         #error-detailed-message {
-          color: #6c7381;
           font-size: 15px;
           font-weight: 400;
           line-height: 22.5px;
           word-wrap: break-word;
         }
-        #error-actions {
-          justify-content: center;
-          align-items: center;
-          gap: 24px;
-          display: inline-flex;
+
+        .error-message {
+          color: #6c7381;
         }
-        #error-action-home {
-          width: 136px;
-          padding-left: 16px;
-          padding-right: 16px;
-          justify-content: center;
-          align-items: center;
-          gap: 8px;
-          display: flex;
+        .similar-features-container {
+          text-align: left;
+          padding: 12px;
+          max-width: 400px;
         }
-        #error-action-report {
-          width: 145px;
-          padding-left: 16px;
-          padding-right: 16px;
-          border-radius: 4px;
-          justify-content: center;
-          align-items: center;
-          gap: 8px;
-          display: flex;
+        .similar-results-header {
+          color: #1a1a1a;
+          font-weight: 500;
+          margin-bottom: 6px;
         }
-
-        #error-action-report a {
-          color: inherit;
+        .feature-list {
+          list-style: none;
+          padding: 0;
+          margin: 0;
+        }
+        .feature-list li {
+          padding: 6px 0;
+        }
+        .feature-list li a {
           text-decoration: none;
+          color: #007bff;
+          font-weight: 500;
+        }
+        .feature-list li a:hover {
+          text-decoration: underline;
+          color: #0056b3;
         }
       `,
     ];
   }
+
   protected render(): TemplateResult {
+    const featureId = getSearchQuery(this.location);
+
+    console.log('hello', featureId);
+
     return html`
-      <div id="error-container">
+      <div
+        id="error-container"
+        style="gap: ${this.similarFeatures.length > 0 ? '32px' : '48px'};"
```

Comment:
We should not inline styles. Instead have a different class for each case. and assign the class appropriately